patches/dpdk/22.11/windows/0013-Use-alarm-to-workaround-vf-uio-no-intr-issue.patch

```diff
@@ -0,0 +1,57 @@
+From ef798145257c6e39682a1e72a203625321e57750 Mon Sep 17 00:00:00 2001
+From: Ric Li <ming3.li@intel.com>
+Date: Thu, 27 Apr 2023 10:39:58 -0700
+Subject: [PATCH] Use alarm to workaround vf uio no intr issue
+
+---
+ drivers/net/iavf/iavf_ethdev.c | 6 +++---
+ drivers/net/iavf/iavf_vchnl.c  | 2 +-
+ 2 files changed, 4 insertions(+), 4 deletions(-)
+
+diff --git a/drivers/net/iavf/iavf_ethdev.c b/drivers/net/iavf/iavf_ethdev.c
+index 3196210f2c..e42a363d8c 100644
+--- a/drivers/net/iavf/iavf_ethdev.c
++++ b/drivers/net/iavf/iavf_ethdev.c
+@@ -2569,7 +2569,7 @@ iavf_dev_init(struct rte_eth_dev *eth_dev)
+ 	struct iavf_adapter *adapter =
+ 		IAVF_DEV_PRIVATE_TO_ADAPTER(eth_dev->data->dev_private);
+ 	struct iavf_hw *hw = IAVF_DEV_PRIVATE_TO_HW(adapter);
+-	struct iavf_info *vf = IAVF_DEV_PRIVATE_TO_VF(adapter);
++	//struct iavf_info *vf = IAVF_DEV_PRIVATE_TO_VF(adapter);
+ 	struct rte_pci_device *pci_dev = RTE_ETH_DEV_TO_PCI(eth_dev);
+ 	int ret = 0;
+ 
+@@ -2637,7 +2637,7 @@ iavf_dev_init(struct rte_eth_dev *eth_dev)
+ 	if (iavf_dev_event_handler_init())
+ 		goto init_vf_err;
+ 
+-	if (vf->vf_res->vf_cap_flags & VIRTCHNL_VF_OFFLOAD_WB_ON_ITR) {
++	if (/*vf->vf_res->vf_cap_flags & VIRTCHNL_VF_OFFLOAD_WB_ON_ITR*/false) {
```

Comment:
can you give a comment to say it's a workaround for UIO driver as IRQ is not working?

Reply:
done